openlibrary/core/lending.py

```diff
@@ -278,16 +276,15 @@ def get_available(limit=None, page=1, subject=None, query=None,
             'sorts': sorts,
         })
     try:
-        request = urllib.request.Request(url=url)
-
         # Internet Archive Elastic Search (which powers some of our
         # carousel queries) needs Open Library to forward user IPs so
         # we can attribute requests to end-users
         client_ip = web.ctx.env.get('HTTP_X_FORWARDED_FOR', 'ol-internal')
-        request.add_header('x-client-id', client_ip)
 
-        content = urllib.request.urlopen(request, timeout=config_http_request_timeout).read()
-        items = simplejson.loads(content).get('response', {}).get('docs', [])
+        response = requests.get(
+            url, headers={"x-client-id": client_ip}, timeout=config_http_request_timeout
+        )
+        items = response.json().get('response', {}).get('docs', [])
```

Comment:
http://staging.openlibrary.org/books/OL13439647M/The_brothers_Karamazov did not raise an exception as it did in (https://sentry.archive.org/sentry/ol-web/issues/3873) on Python 2 but it was not available for borrow.